### PR TITLE
Move customs module to content/modules

### PR DESCRIPTION
**Pull request by duythien**

### CODE REVIEW

**Comment by duythien** [comment]
@sergeyklay Take look my commit. I think we should move custom modules into contents/module directory

File: bootstrap/helpers.php [hunk 1]
```diff
@@ -178,3 +178,15 @@ function container()
...
     }
 }
+if (!function_exists('content_modules_path')) {
+    /**
+     * Get the modules path.
+     *
+     * @param  string $path
+     * @return string
+     */
+    function content_modules_path($path = '')
```
Comment (sergeyklay) [inline bootstrap/helpers.php]: I suggest to use `modules_path()`

Comment (sergeyklay) [inline bootstrap/helpers.php]: Oh, nwm

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
Looks good to me

**Pull request merged**